bcda/main.go

```diff
@@ -48,6 +48,7 @@ import (
 )
 
 func init() {
+	log.SetupLoggers()
```

Comment:
Do you think we can call `log.SetupLoggers()` from either here or cli.go instead of both places, since this main.go immediately calls cli.go's setupApp() ?